src/Diagnostics.ModelsAndUtils/Models/DataTableResponseObject.cs

```diff
@@ -125,14 +125,11 @@ public static DataTableResponseObject ToDataTableResponseObject(this DataTable t
                 columns.Add(new DataTableResponseColumn() { ColumnName = col.ColumnName, DataType = col.DataType.ToString().Replace("System.", "") });
             }
 
-            var rows = new dynamic[table.Rows.Count, table.Columns.Count];
+            var rows = new dynamic[table.Rows.Count][];
```

Comment:
new dynamic[table.Rows.Count] [table.Columns.Count]??

Reply:
You cannot do this for C# Jagged arrays. The compiler doesn't allow this.

https://stackoverflow.com/a/4384236/1900166